GTFO-API/API/AssetAPI.cs

```diff
@@ -33,6 +33,19 @@ public static class AssetAPI
         /// </summary>
         public static event Action OnImplReady;
 
+        /// <summary>
+        /// Checks if an asset is already registered in the <see cref="AssetAPI"/>
+        /// </summary>
+        /// <param name="name">Name of the asset to check</param>
+        /// <returns>Whether the asset is registered or not</returns>
+        public static bool IsAssetRegistered(string name)
```

Comment:
Just name this as `ContainsKey`, if possible, for consistency and brevity

Reply:
I personally dislike the format of `AssetAPI.ContainsKey(string name)` it's too vague compared to methods like `GetLoadedAsset`